System.Compiler/Nodes.cs

```diff
@@ -16245,6 +16245,20 @@ public RequiresList Validations
       get { return this.validations; }
       set { this.validations = value; }
     }
+
+    /// <summary>
+    /// Returns true if current contract node has at least one assertion in it
+    /// (like pres/posts etc).
```

Comment:
Yep, actually this would be more accurate statement. Because if a method contains another assertion like `Contract.Assert`/`Contract.Assume` this method will not return true.